src/components/Header.js

```diff
@@ -1,11 +1,25 @@
-
 const Header = () => {
+  const headerStyle = {
+    padding: "20px 0",
+    lineHeight: "1.5em",
+  };
   return (
-    <header>
-      <h1>todos</h1>
+    <header style={headerStyle}>
+      <h1
+        style={{
+          fontSize: "6rem",
+          fontWeight: "600",
+          marginBottom: "2rem",
+          lineHeight: "1em",
+          color: "#ececec",
+          textTransform: "lowercase",
+          textAlign: "center",
+        }}

```

Comment:
It would be better to use a CSS file for the styling so your JS file ca work more faster